code/modules/reagents/chemistry/recipes/special.dm

```diff
@@ -171,22 +171,6 @@ GLOBAL_LIST_INIT(medicine_reagents, build_medicine_reagents())
 			return food_reagent_ids
 	return ..()
 
-///Random recipe for meme chem metalgen. Always requires wittel and resets every 3 days
-/datum/chemical_reaction/randomized/metalgen
```

Comment:
did you remember to remove this recipe from the paper wizard space ruin as well?